File: src/tool/spring-boot-generator/output/spider/db/domainInfo.js
let domainInfo = {
	name: "domainInfo",
	description: "domain信息",
	columns: {
		id: {
			name: "id",
			description: "编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		domainId: {
			name: "domainId",
			description: "网站id",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		domain: {
			name: "domain",
			description: "网站名字",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		maxSpeed: {
			name: "maxSpeed",
			description: "最大速度",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		needCookie: {
			name: "needCookie",
			description: "需要cookie",
			nullable: false,
			type: {
				name: "smallint",
				length: null
			}
		},
		remark: {
			name: "remark",
			description: "备注",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		encoding: {
			name: "encoding",
			description: "编码",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		httpProtocal: {
			name: "httpProtocal",
			description: "协议类型",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		type: {
			name: "type",
			description: "网站类型",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		}
	}
};

let columnsArray = [
	"id",
	"domainId",
	"domain",
	"maxSpeed",
	"needCookie",
	"remark",
	"encoding",
	"httpProtocal",
	"type"
];

function create() {
  return {};
}

domainInfo.columnsArray=columnsArray;
domainInfo.create=create;
module.exports=domainInfo;

File: src/tool/spring-boot-generator/output/spider/spider/src/main/java/com/spider/pojo/entity/DomainInfo.java
package com.spider.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;



@ApiModel("domain信息")
@Getter
@Setter
public class DomainInfo  {

    @ApiModelProperty("网站名字")
    private String domain;

    @ApiModelProperty("网站id")
    private Integer domainId;

    @ApiModelProperty("编码")
    private String encoding;

    @ApiModelProperty("协议类型")
    private String httpProtocal;

    @ApiModelProperty("编号")
    private Integer id;

    @ApiModelProperty("最大速度")
    private Integer maxSpeed;

    @ApiModelProperty("需要cookie")
    private Boolean needCookie;

    @ApiModelProperty("备注")
    private String remark;

    @ApiModelProperty("网站类型")
    private String type;

}